content/post/thriving-oss-community.md

```diff
@@ -0,0 +1,138 @@
+author = "cblake"
++++
+
+ 
+Building a community of developers was one of the key motivations that led Wallaroo Labs to open-source our distributed data engine, Wallaroo.
+ 
+But it’s not always easy. There are millions of public, open-source repositories on GitHub. How do you stand out from the crowd and build a thriving developer community around your project?
+ 
+The plan we developed to grow our community was the same as the one we used with our open-source code: put our best foot forward, then learn and adjust as we go along.
+ 
+In this post, I will share some of the initial ideas, tactics, and approaches that we are employing and believe will drive the success of our community.
+ 
+## Set Goals and Measure Results
+ 
+The beauty of GitHub is that it brings together developers from all over the world. That is a huge opportunity! How will you benefit from this collection of potential advisors, critics, contributors, and customers?
+ 
+Start by considering your own repository. What are your critical metrics, and how can you measure them?
+ 
+If you don’t know what you hope to achieve, then what’s the point of opening up your code to the public at all? You might as well keep it closed. Identify your main goal. Is it software downloads?  Getting developers to contribute to your project? Adding names to your developer mailing list?
+ 
+All are reasonable objectives, but they only help you if you measure their results.
+ 
+It might take some setup and development efforts to put a results-tracking mechanism in place. Even then, not everything you want to achieve will be measurable. Be sure you get your head around measurement early in the process, so you can create a plan and adapt as things change.
+ 
+At Wallaroo Labs, one of the most important metrics is the "Unique clones" number (included in the Traffic Statistics section of the GitHub repo reports). This metric gives us a good idea of our daily download activity.  We also log all of the GitHub repository statistics on a regular basis. Stars, Forks, Issues, Pull Requests – these statistics give us usage “signals."
+ 
+Estimating future marketing activities and product evolution allows us to put a straw-man plan in place, giving us a way to measure our progress against any plan.
+ 
+## Be Transparent with Developers
+ 
+Developers don’t appreciate being misled. We believe that being open and honest about the state of our software and our progress will help build loyalty in our community.
+ 
+And we practice what we preach. We know that our product isn’t perfect, our mission, to democratize data processing by removing the barriers of complexity, cost, performance, and language support, will keep our team busy for the foreseeable future as we try to get it there. In the meantime, we share our shortcomings. Open-source means open-obstacle too.
+
+Here’s an example.
```

Comment:
Having this as a standalone paragraph feels a little strange. It almost feels like something is missing between this and the next paragraph. I think it would make more sense if this sentence was put at the beginning of the next paragraph.